examples/tutorials/nb_python/coordinate_frame_tutorial.py

```diff
@@ -0,0 +1,308 @@
+# ---
+# jupyter:
+#   accelerator: GPU
+#   colab:
+#     collapsed_sections: []
+#     name: Coordinate Frame Conventions
+#     provenance: []
+#   jupytext:
+#     cell_metadata_filter: -all
+#     formats: nb_python//py:percent,colabs//ipynb
+#     notebook_metadata_filter: all
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Coordinate Frame Conventions
+# A review of coordinate frame conventions in Habitat
+#
+# Don't have the patience to install and run this Colab? A cached version with all output images is available [here](https://colab.research.google.com/gist/eundersander/c7cfe6243149efb073ccf75b262bba31/coordinate-system-tutorial.ipynb)!
+
+# %% [markdown]
+# # Installation
+
+# %%
+# !curl -L https://raw.githubusercontent.com/facebookresearch/habitat-sim/master/examples/colab_utils/colab_install.sh | NIGHTLY=true bash -s
+
+# %%
+# @title Colab setup, imports, and utilities :{ display-mode: "form" }
+import os
+import sys
+
+import git
+import magnum as mn
+from PIL import Image
+
+import habitat_sim
+
+# %cd /content/habitat-sim
+repo = git.Repo(".", search_parent_directories=True)
+dir_path = repo.working_tree_dir
+# %cd $dir_path
+data_path = os.path.join(dir_path, "data")
+is_colab = "google.colab" in sys.modules
+# images will be either displayed in the colab or saved as image files
+if is_colab:
+    is_colab = True
+    import IPython.display
```

Comment:
is_colab != is_ipython. You can run these notebooks locally too.

Reply:
>is_colab != is_ipython
I don't claim these are equivalent. The intention of this code is to use `IPython.display` for the case of google colab and save to an image file otherwise.

Are you suggesting that I use `IPython.display` for the case of running the notebook locally?

Reply:
yes

Reply:
There are more generic way of detecting if an environment is an ipython and those should be used instead.